src/levenshtein.cr

```diff
@@ -13,11 +13,11 @@ module Levenshtein
   def self.distance(string1 : String, string2 : String) : Int32
     return 0 if string1 == string2
 
-    s = string1.chars
-    t = string2.chars
+    s = string1.to_unsafe
+    t = string2.to_unsafe
```

Comment:
I think it should be possible to use `String#each_char_with_index` instead?

Reply:
What slice do you mean? This is a pointer.

Reply:
Well doesn't matter too much. They are very similar.